scripts/provision_db2_devc.rb

```diff
@@ -0,0 +1,88 @@
+require_relative 'utilities'
+require_relative 'commodities'
+require 'yaml'
+
+def provision_db2_devc(root_loc)
+  puts colorize_lightblue('Searching for db2_devc initialisation SQL in the apps')
+
+  # Load configuration.yml into a Hash
+  config = YAML.load_file("#{root_loc}/dev-env-config/configuration.yml")
+  return unless config['applications']
+
+  database_initialised = false
+
+  config['applications'].each do |appname, _appconfig|
+    # To help enforce the accuracy of the app's dependency file, only search for init sql
+    # if the app specifically specifies db2_devc in it's commodity list
+    next unless File.exist?("#{root_loc}/apps/#{appname}/configuration.yml")
+    next unless commodity_required?(root_loc, appname, 'db2_devc')
+
+    # Load any SQL contained in the apps into the docker commands list
+    if File.exist?("#{root_loc}/apps/#{appname}/fragments/db2-devc-init-fragment.sql")
+      database_initialised = process_db2_devc_fragment(root_loc, appname, database_initialised)
+    else
+      puts colorize_yellow("#{appname} says it uses DB2 Developer C but doesn't contain an init SQL file.
+          Oh well, onwards we go!")
+    end
+  end
+end
+
+def process_db2_devc_fragment(root_loc, appname, database_initialised)
+  result = database_initialised
+  puts colorize_pink("Found some in #{appname}")
+  if commodity_provisioned?(root_loc, appname, 'db2_devc')
+    puts colorize_yellow("DB2 Developer C has previously been provisioned for #{appname}, skipping")
+  else
+    unless database_initialised
+      init_db2_devc
+      result = true
+    end
+    init_sql(root_loc, appname)
+  end
+  result
+end
+
+def init_sql(root_loc, appname)
```

Comment:
We have been burnt with Postgres 9.6 with duplicate method names, although this is only called from other methods within this module, it's probably safest if it is renamed to not clash with the other db2 provision file
```suggestion
def init_db2_devc_sql(root_loc, appname)
```